src/NUnitFramework/framework/Internal/Builders/DefaultSuiteBuilder.cs

```diff
@@ -176,7 +176,7 @@ private bool HasArguments(IFixtureBuilder attr)
             // Only TestFixtureAttribute can be used without arguments
             var temp = attr as TestFixtureAttribute;
 
-            return temp == null || temp.Arguments.Length > 0 || temp.TypeArgs.Length > 0;
+            return temp == null || (temp.Arguments != null && temp.Arguments.Length > 0) || (temp.TypeArgs != null && temp.TypeArgs.Length > 0);
         }
```

Comment:
@jnm2 It would have to be a postcondition in every constructor in that case. Maybe simpler to add a check to the tests of the attribute itself.